sport/app/football/views/wallchart/knockoutSpider.scala.html

```diff
@@ -50,7 +50,7 @@
                                 football-round--@matches.length-matches
                                 football-round--final
                                 @if(knockoutStage.isActiveRound(round)){football-round--active}
-                                @round.name.filter(n => n == "3rd/4th Play-Offs").map{ name => football-round--playoff}">
+                                @round.name.filter(n => n == "3rd/4th Play-Offs" || n == "3rd/4th Place Play-Off").map{ name => football-round--playoff}">
```

Comment:
It might be worth considering if we should implement a fuzzier filter here. Depends how likely we think it is that the data will change from PA in the future.

In truth, I think the men's strings have been pretty stable so we can probably assume the women's will either:
* remain as is
* switch to the same as the men's

This means probably not a priority, but up to you what you think.

Reply:
I will make a surprise card called "World cup spider improvement. Do not open before world cup 2022" :)
Thanks Adam (^_^)